app/components/dashboard/all_collections_row_component.html.erb

```diff
@@ -0,0 +1,10 @@
+<tr>
+  <td><%= link_to collection.head.name, collection %><% if draft? %><strong><i> <%= draft_label %></i></strong><% end %></td>
```

Comment:
How about putting all this HTML and the conditional into a method in the component?  so just `<= draft_label %>` here and then `draft_label` can `return unless draft?`.